modules/ROOT/pages/dos-policy.adoc

```diff
@@ -42,48 +47,50 @@ Violations to a rate limiting, or SLA based, or Ping Federate policy at the API
+* *Limit its messages*: This rule restricts the specified client's message rate until the end of the time window you configure (for the rest of the specified time window or indefinitely).
+* *Block all messages*: The policy rejects all the client's message requests until the end of the time window you configure (for the rest of the specified time window or indefinitely).
 
-After the interval of your window of time expires, the policy does not enforce shape-interval nor block-interval. +
-Within each window, you can configure the number of errors that you want to allow, within this time window. This number of errors is a DoS count. You can configure up to three DoS windows for each type of DoS error. Within each window, the policy can apply one of the DoS actions to the client’s message requests, whenever your configured DoS count is met. +
-For example, you can configure a 60 second window (1 minute interval), and configure the policy so that if 2 protocol error happens within that time window ("2" is then your DoS count), reject all further requests from this client (apply a Block-Forever action).
+After the specified interval of time expires, the policy does not enforce shape- or block-interval. +
+Within each window, you can configure the number of errors that you want to allow within the time window. This number of errors is a DoS count. You can configure up to three DoS windows for each type of DoS error. Within each window, the policy can apply one of the DoS actions to the client’s message requests whenever your configured DoS count is met. +
+For example, you can configure a 60 second window (1 minute interval), and configure the policy so that if 2 protocol errors happen within that time window ("2" is then your DoS count), all further requests from the client are rejected (apply a Block-Forever action).
 
-You can configure up to three different windows. Each window corresponds to escalation levels. *Window 1* is the lowest escalation level and *Window 3* is the highest. +
-When configuring a window higher than *Window 1*, you can set up the interval for those windows based on the interval of the preceding window (Window 1 for Window 2, and Window 2 for Window 3). +
-Following the example above, you can set up Window 2 to be two intervals of Window 1, which means that if Window 1 has an interval of 60 seconds, then Window 2 will have a 120 seconds interval. +
-Additionally, you can configure a different DoS count for each window, which introduces the concept of thresholds. +
-Each window has a different threshold:
+You can configure up to three different rules for each error. Each rule corresponds to escalation levels. *Rule A* is the lowest escalation level and *Rule C* is the highest escalation level. +
+When configuring a rule with a higher escalation level than *Rule A*, you can set up the interval for those rules based on the interval of the preceding rule (Rule A for Rule B, and Rule B for Rule C). +
```

Comment:
Please replace "set up" with just "set" -- "up" isn't needed and inhibits readability